inventory/pom.xml

```diff
@@ -156,9 +156,6 @@
                 <groupId>net.wasdev.wlp.maven.plugins</groupId>
                 <artifactId>liberty-maven-plugin</artifactId>
                 <configuration>
-                    <install>
-                        <runtimeUrl>https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/wasdev/downloads/wlp/beta/wlp-beta-2018.5.0.0.zip</runtimeUrl>
```

Comment:
Isn't the runtime URL still necessary when running locally outside of minikube?  Maybe we can switch it to 18.0.0.2 instead.

Reply:
It is getting 18.0.0.2 by default when I do `mvn install` 

`[INFO] Getting: https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/wasdev/downloads/wlp/18.0.0.2/wlp-webProfile7-18.0.0.2.zip
[INFO] To: /Users/Hemankita.Perabathini@ibm.com/.m2/repository/wlp-cache/18.0.0_02/wlp-webProfile7-18.0.0.2.zip`

Reply:
That's good. I'm guessing it's using the parent pom so it should be pulling in the latest if that is the case 👍